src/ostorlab/assets/api_schema.py

```diff
@@ -5,6 +5,9 @@
 
 from ostorlab.assets import asset
 
+PROTO_FIELD = "api_schema"
+CLASS_NAME = "API Schema"
```

Comment:
If these constants are specific to the class, consider moving them inside the class definition and prefix them with an underscore to indicate that they are internal.

Reply:
If PROTO_FIELD is used only once, I might consider directly using the string "api_schema" in the property method. but If the constant might be used in other parts of the module in the future, keeping it as a module-level constant is beneficial.

Reply:
I used a const because of this comment https://github.com/Ostorlab/oxo/pull/756#discussion_r1678384808